packages/app/src/app/pages/Sandbox/Editor/Header/Actions.tsx

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import { useOvermind } from 'app/overmind';
+
+import { UserMenu } from 'app/pages/common/UserMenu';
+import Tooltip from '@codesandbox/common/lib/components/Tooltip';
+
+import { Stack, Avatar, Button } from '@codesandbox/components';
+import css from '@styled-system/css';
+import {
+  ReloadIcon,
+  PreferenceIcon,
+  LikeIcon,
+  EmbedIcon,
+  ForkIcon,
+} from './icons';
+
+const TooltipButton = ({ tooltip, ...props }) => (
+  <Tooltip content={tooltip}>
+    <Button {...props} />
+  </Tooltip>
+);
+
+export const Actions = () => {
+  const {
+    actions: {
+      modalOpened,
+      editor: { likeSandboxToggled, forkSandboxClicked },
+      explore: { pickSandboxModal },
+    },
+    state: {
+      hasLogIn,
+      updateStatus,
+      user,
+      editor: {
+        currentSandbox: { id, owned, title, description, likeCount, userLiked },
+      },
+    },
+
+    actions: { signInClicked },
+  } = useOvermind();
+
+  const handleSignIn = async () => {
+    await signInClicked({ useExtraScopes: false });
+  };
+
+  let primaryAction;
+  if (!hasLogIn) primaryAction = 'Sign in';
+  else primaryAction = owned ? 'Embed' : 'Fork';
+
+  return (
+    <Stack align="center" gap={1} css={{ button: { width: 'auto' } }}>
+      {updateStatus === 'available' && (
+        <TooltipButton
```

Comment:
This should be green or something like it s now to call out attentio

Reply:
The idea was not to call too much attention to it 🤷‍♀ because it isn't the core thing you're here for

Reply:
The issue is that if people don't click they may get unrelated bugs :/

Reply:
important problem, bad solution?